src/icons/list-arrows.svg

```diff
@@ -1,10 +1,10 @@
 <svg xmlns="http://www.w3.org/2000/svg" width="36" height="36" viewBox="0 0 36 36">
-  <g fill-rule="evenodd">
-    <polygon points="11 11 33 11 33 8.999 11 8.999"/>
-    <polygon points="11 19 33 19 33 17 11 17"/>
-    <polygon points="11 27 33 27 33 25 11 25"/>
-    <polygon points="5.091 7 8 10 5.091 13 4 11.875 5.818 10 4 8.125"/>
-    <polygon points="5.091 15 8 18 5.091 21 4 19.875 5.818 18 4 16.125"/>
-    <polygon points="5.091 23 8 26 5.091 29 4 27.875 5.818 26 4 24.125"/>
+  <g fill="none" fill-rule="evenodd">
+    <polygon fill="#000" points="11 11 33 11 33 8.999 11 8.999"/>
```

Comment:
@nileshgulia1 I wonder why is it filled with `#000`

Reply:
Ok, it seems broken in master https://github.com/plone/volto/blob/master/src/icons/verified.svg vs the one from your pr https://github.com/plone/volto/blob/edbee4f2366aa94b211ebbcbeadbba6ad64a1436/src/icons/verified.svg